cypress/integration/happy-path-with-accessibility.js

```diff
@@ -34,20 +34,27 @@ function violationsCallBack(violations) {
 
 function checkAccessibility() {
   if(Cypress.config('accessibility')) {
+    cy.injectAxe();
```

Comment:
The PR's description is not clear why was this needed and what this actually solves? Could you add these notes on please?

Reply:
As suggested in the description and to re-iterate, we need to inject Axe as this step will fail otherwise each time we call it

Reply:
The description just says `We need to inject axe before each time we call it` - it says what you did but not why you did it hence my question above. :)

Reply:
Failure is implied if we need it